docs/Tutorials/CrossCompilationSetup/ArmLinuxTutorial.md

```diff
@@ -0,0 +1,36 @@
+# F´ Running on ARM Linux Tutorial
+
+For this tutorial, the assumption is that the ARM Linux machine is available on the network, is running SSH, and the username, password, device address, and host address are known. Without this configuration, users should skip to the next section of the tutorial.
+
+First, in a terminal upload the software to hardware platform. This is done with:
+
+```sh
+# For ARM 64-bit hardware
+# In: Deployment Folder
+scp -r build-artifacts/aarch64-linux <username>@<device-address>:[NameOfBinary]
+
+# For ARM 32-bit hardware
+# In: Deployment Folder
+scp -r build-artifacts/arm-hf-linux <username>@<device-address>:[NameOfBinary]
+```
+> Users must fill in the username and device address above.
+
+Next run the F´ GDS without launching the native compilation (`-n`) and with the 
+dictionary from the build above (`--dictionary ./build-artifacts/<platform name>/<.xml document>`).
+
+```sh
+# For in-person workshops and ARM 64-bit hardware
+# In: Deployment Folder
+fprime-gds -n --dictionary build-artifacts/aarch64-linux/dict/<App Dictionary>.xml
+
+# For ARM 32-bit hardware
+# In: Deployment Folder
+fprime-gds -n --dictionary build-artifacts/aarch64-linux/dict/<App Dictionary>.xml
+```
+
```

Comment:
We should add a step noting one should also SSH into the hardware to run the binary.

Reply:
Added step for SSHing into device and running the executable